app/controllers/api/api_controller.rb

```diff
@@ -48,5 +48,26 @@ def status
       end
       render json: response, status: :ok
     end
+
+    private
+
+    def filter_timestamp_range(records)
+      from = params.dig(:create_date_start)
```

Comment:
use this format for params: filter: { created_at: { from: '', to: '' }, updated_at: { from: '', to: '' } }